src/forms/assetForm.tsx

```diff
@@ -819,14 +819,15 @@ class AssetForm extends React.Component<AssetFormProps, AssetFormState> {
   handleSiteSelect(site: DatacenterObject) {
     const clearedNetworkConnections = this.getClearedNetworkConnections();
     const clearedPowerConnections = this.getClearedPowerSelections();
+
     const newValues = updateObject(this.state.values, {
       rack: undefined,
-      rack_position: null,
       chassis: undefined,
-      chassis_slot: null,
       power_connections: clearedPowerConnections,
       network_connections: clearedNetworkConnections,
     });
+    delete newValues.rack_position;
+    delete newValues.chassis_slot;
 
```

Comment:
for string values, if you set it to null or undefined the form just ignores it and keeps the old value

Reply:
OOOOOO. thx